### DISCUSSION TITLE
Stylizing graph

### DISCUSSION THREAD

**Discussion by loic-lopez** [beyondcode/laravel-er-diagram-generator#110] [Show and tell]
Hello,

Someone knows good settings for generate a beautiful diagram?

Thanks for reply?

**Comment by diogogomeswww** [comment]
Hi @loic-lopez,

You can style it on the config settings:
```
    /*
     * Here you can define all the available Graphviz attributes that should be applied to your graph,
     * to its nodes and to the edge (the connection between the nodes). Depending on the size of
     * your diagram, different settings might produce better looking results for you.
     *
     * See http://www.graphviz.org/doc/info/attrs.html#d:label for a full list of attributes.
     */
    'graph' => [
        'style' => 'filled',
        'bgcolor' => '#F7F7F7',
        'fontsize' => 12,
        'labelloc' => 't',
        'concentrate' => true,
        'splines' => 'polyline',
        'overlap' => false,
        'nodesep' => 1,
        'rankdir' => 'LR',
        'pad' => 0.5,
        'ranksep' => 2,
        'esep' => true,
        'fontname' => 'Helvetica Neue'
    ],
```

Publish the config options:
`php artisan vendor:publish --provider=BeyondCode\\ErdGenerator\\ErdGeneratorServiceProvider` 

And then change whatever you need or want.

**Comment by loic-lopez** [comment]
Hi, thanks for reply but i just ask for the best preset for generate a diagram.

**Comment by loic-lopez** [comment]
@diogogomeswww like in the readme:
![image](https://user-images.githubusercontent.com/17724398/49044941-58656680-f19d-11e8-92e3-09b67dcf952d.png)

**Comment by ahren-condos-ca** [comment]
I second this question. its confusing that the example in the readme looks nothing like the default settings in the `erd-generator.php` config file

**Comment by swatty007** [comment]
Same here. Would be nice to have that config file included as well.

**Comment by Zanark** [comment]
Do you have the config for the diagram with the entity members.
